ee/se/modules/380-metallb/crds/metalloadbalancerclasses.yaml

```
@@ -0,0 +1,93 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  labels:
+    module: metallb
+    backup.deckhouse.io/cluster-config: "true"
+  name: metalloadbalancerclasses.network.deckhouse.io
+spec:
+  group: network.deckhouse.io
+  names:
+    kind: MetalLoadBalancerClass
+    listKind: MetalLoadBalancerClassList
+    plural: metalloadbalancerclasses
+    singular: metalloadbalancerclass
+  scope: Cluster
+  versions:
+    - name: v1alpha1
+      schema:
+        openAPIV3Schema:
+          description: MetalLoadBalancerClass is the Schema for the metalloadbalancerclasses API
+          properties:
+            apiVersion:
+              type: string
+            kind:
+              type: string
+            metadata:
+              type: object
+            spec:
+              properties:
+                isDefault:
+                  type: boolean
+                  x-doc-examples: [true]
+                  description: |-
+                    Whether MetalLoadBalancerClass is selected for all services by default.
+                type:
+                  type: string
+                  enum:
```

```suggestion
                  enum:
```
It is proper to have an enum with a single value?

---

Fixed it